docs/Plugins.md

```diff
@@ -11,6 +11,34 @@ You already see in the [getting started](https://github.com/fastify/fastify/blob
 fastify.register(plugin, [options])
 ```
 
+<a name="plugin-options"></a>
+### Plugin Options
+The optional `options` parameter for `fastify.register` gets supports a predefined set of options that Fastify itself will use. This options object will also be passed to the plugin upon invocation. The currently supported list of Fastify specific options is:
+
++ [`prefix`](https://github.com/fastify/fastify/blob/master/docs/Plugins.md#route-prefixing-options)
```

Comment:
There is also https://github.com/fastify/fastify/blob/4d61fc4c5b5063a9370d37d8fc52c9038ac0e620/docs/Routes.md#custom-log-level

Reply:
I knew something was missing. It should have been in this document.